### ISSUE TITLE
lookup.js Changes I have tested

### ISSUE THREAD

**Issue by StarfighterJ** [pparedes1/alexa-lgtv-remote#4]
I tried to pull but I either don't know how or it won't let me but here is a copy and Paste of it:

const appDictionary = {
    'tv': 'com.webos.app.livetv',
    'amazon': 'amazon',
    'netflix': 'netflix',
    'google': 'googleplaymovieswebos',
    'youtube': 'youtube.leanback.v4',
    'hulu': 'hulu',
    'sling': 'com.movenetworks.app.sling-tv-sling-production',
    'live': 'com.webos.app.livetv',
    'vudu': 'vudu',
    'crackle': 'crackle',
    'music': 'com.webos.app.music',
    'accessibility': 'com.webos.app.accessibility',
//The above work, below will ok but do not work on the TV I tried both com.webos.app and webos at the end.
    'channelplus': 'channelpluswebos',
    'fandangonow': 'fandangonow',
    'lgstore': 'lgcontentstorewebos',
    'search': 'searchwebos',
    'webbrowser': 'webbrowserwebos',
    'userguide': 'userguidewebos',
    'deviceconnector': 'deviceconnectorwebos',
    'guide': 'guidewebos',
    'photo': 'photo&videowebos',
    'tvscheduler': 'tvschedulerwebos',
    'screenshare': 'screensharewebos',
    'notifications': 'notificationswebos',

**Comment by pparedes1** [comment]
I went into another node package and pulled the apps installed on my tv, below are some of the items you are missing from the JSON:

'channelplus': 'channelplus.us',
'fandango': 'com.mgo.mgowebos',
'plex': 'cdp-30',
'lgcontent': 'com.webos.app.discovery',
'search': 'com.webos.app.voice',
'guide': 'com.webos.app.tvguide',
'pictures': 'com.webos.app.photovideo',
'gallery': 'com.webos.app.igallery',
'tvscheduler': 'com.webos.app.scheduler',
'screenshare': 'com.webos.app.miracast',
'notifications':'com.webos.app.notificationcenter',
'guide':'com.webos.app.tvguide',
'deviceconnector':'com.webos.app.connectionwizard',
'webbrowser':'com.webos.app.browser'

Hope these help, let me know if they don't work. I'll add a few more of these in an upcoming commit. Will try the HDMI inputs tomorrow.

**Comment by StarfighterJ** [comment]
Nice find I'll test tomorrow fyi you left these two out on look up file.
'music': 'com.webos.app.music',
'accessibility': 'com.webos.app.accessibility',

**Comment by pparedes1** [comment]
thanks for catching updated the commit (actually cleaned up my history) with those two items added

**Comment by pparedes1** [comment]
found the culprit on switching inputs, the regular expression would replace the numbers in hdmi1 to hdmi and fail in lookup.js. Not sure why voydz would eliminate numbers, but I modified the regular expression from `app = app.replace(/[^a-z]/gi, '');` to `app = app.replace(/[^a-z1-9]/gi, '');` and published. I was able to test with asking app to switch input to 'something' and it worked. 

Let me know if you are still having trouble or I'll close this one out.
